.github/workflows/aws-integration-test.yml

```diff
@@ -0,0 +1,138 @@
+permissions:
+  contents: read
+name: LocalStack AWS Integration Test
+on:
+  workflow_call:
+  workflow_dispatch:
+jobs:
+  aws_modular_integration_test:
+    name: aws_modular_integration_test
+    runs-on: ubuntu-latest
+    services:
+      setup-localstack-service:
+        image: localstack/localstack
+        ports:
+          - '4566:4566'
+        env:
+          SERVICES: 's3,iam,sts'
+          DEFAULT_REGION: eu-north-1
+          FORCE_NONINTERACTIVE: 1
+          AWS_ACCESS_KEY_ID: test
+          AWS_SECRET_ACCESS_KEY: test
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v2
+        with:
+          terraform_version: 1.3.4
```

Comment:
wondering if this shouldn't be a bit higher?

Reply:
I was wondering the same thing. I did receive a suggestion while running it suggesting to upgrade to 1.7.x, but kept it as `1.3.4` for consistency as that was the version used in `ci.yml` .

Reply:
Let's go to 1.6.0.

Reply:
I'll test it locally and make sure it doesn't break anything, and will repush with the other suggested changes (removing the cleanup steps). Thank you!

Reply:
All worked well. I updated the documentation as well to reflect our changes.